shapes/triangle.py

```diff
@@ -0,0 +1,57 @@
+"""
+Author: Dmytro Iurchuk
+File triangle.py
+Purpose: Defines a Triangle object, inherited from the abstract class Shape.
+"""
+
+from math import sqrt
+from heapq import nsmallest
+from shapes.shape import Shape
+
+class Triangle(Shape):
+    """
+    Represents a Triangle shape, and contains three sides length values
+    and number of decimal places to use when computing values.
+    """
+    decimal_places = 2
+
+    def __init__(self, a, b, c):
+        self.a = int(a)
+        self.b = int(b)
+        self.c = int(c)
+
+    def area(self):
+        """
+        Compute the area using Heron's formula sqrt(s * (s - a) * (s - b) * (s - c))
+        where s is semi-perimeter and a, b and c - corresponding sides
+        """
+        s = self.perimeter() / 2
+        return round(sqrt(s * (s - self.a) * (s - self.b) * (s - self.c)), self.decimal_places)
+
+    def perimeter(self):
+        """
+        Compute the perimeter using the formula a + b + c
+        """
+        return self.a + self.b + self.c
+
+    def is_eqilateral(self):
+        """
+        Determine if the triangle is a equilateral by comparing the length
+        of each side for equality. If all three sides are equal - triangle called equilateral.
+        """
+        return (self.a == self.b) and (self.a == self.c) and (self.b == self.c)
```

Comment:
It would be enough to just say:
`(self.a == self.b) and (self.a == self.c)`

If a==b and a==c, then we know b==c without needing to test.